package.json

```diff
@@ -10,7 +10,8 @@
     "axios": "^0.19.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.3.0"
+    "react-scripts": "3.3.0",
+    "typescript": "^3.7.4"
```

Comment:
Were we going to use ts for this project? I'm confused because typescript package is added but the new components are js files.

Reply:
I wasn't my intention to use Typescript; I'll remove that dependency in the next PR.